From 200fc212408ae38b86f6d42432c6cdcdef0b969b Mon Sep 17 00:00:00 2001
Message-Id: <200fc212408ae38b86f6d42432c6cdcdef0b969b.1350297259.git.jdenemar@redhat.com>
From: Marcelo Cerri <mhcerri@linux.vnet.ibm.com>
Date: Wed, 10 Oct 2012 10:15:07 +0200
Subject: [PATCH] doc: update description about user/group in qemu.conf

https://bugzilla.redhat.com/show_bug.cgi?id=860519

As a side effect of changes in the functions virGetUserID and
virGetGroupID, the user and group configurations for DAC in qemu.conf
are now able to accept both names and IDs, supporting a leading plus
sign to ensure that a numeric value will not be interpreted as a name.

This patch updates the comments in qemu.conf, including a description of
this new behavior.
(cherry picked from commit 60dea2c6bff0d3ee0df3bda9c8521bdfc42407f3)
---
 src/qemu/qemu.conf | 19 +++++++++++++++++--
 1 file changed, 17 insertions(+), 2 deletions(-)

diff --git a/src/qemu/qemu.conf b/src/qemu/qemu.conf
index 18105ca..dd853c8 100644
--- a/src/qemu/qemu.conf
+++ b/src/qemu/qemu.conf
@@ -180,10 +180,25 @@
 # guests will be blocked. Defaults to 0.
 #security_require_confined = 1
 
-# The user ID for QEMU processes run by the system instance.
+# The user for QEMU processes run by the system instance. It can be
+# specified as a user name or as a user id. The qemu driver will try to
+# parse this value first as a name and then, if the name doesn't exist,
+# as a user id.
+#
+# Since a sequence of digits is a valid user name, a leading plus sign
+# can be used to ensure that a user id will not be interpreted as a user
+# name.
+#
+# Some examples of valid values are:
+#
+#       user = "qemu"   # A user named "qemu"
+#       user = "+0"     # Super user (uid=0)
+#       user = "100"    # A user named "100" or a user with uid=100
+#
 #user = "root"
 
-# The group ID for QEMU processes run by the system instance.
+# The group for QEMU processes run by the system instance. It can be
+# specified in a similar way to user.
 #group = "root"
 
 # Whether libvirt should dynamically change file ownership
-- 
1.7.12.3

